redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  labels:
+    app: redirector-k8s-io
+    version: dirty
+spec:
+  revisionHistoryLimit: 2
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxSurge: 4
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: redirector-k8s-io
+      component: txtdirect
+  template:
+    metadata:
+      labels:
+        app: redirector-k8s-io
+        component: txtdirect
+    spec:
+      securityContext:
+        fsGroup: 1337
+        runAsNonRoot: true
+      containers:
+      - name: txtdirect
+        image: gcr.io/txtdirect-223710/txtdirect:dev-7c870d0
+        securityContext:
+          runAsUser: 1337
+          readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop: ["all"]
+        command: [ "/txtdirect", "--log=stdout", "--conf=/config/caddyfile" ]
+        resources:
+          requests:
+            memory: "16Mi"
+            cpu: "25m"
+          limits:
+            memory: "32Mi"
```

Comment:
don't set memory limit > request

Reply:
That got juggled I think. Doesn't make sense as it is here. Will fix it.